rulesdoc.md

```diff
@@ -4167,7 +4181,7 @@ You add 1 superior die when making Enchantment rolls. Enchantments you make last
 #### Mana Font
 *Requires 2 Intelligence*
 
-While in combat you may spend 3 AP to restore spell slots with a combined level equal to 1/3rd of your level, rounded up. After use, cannot be used again until taking a rest.
+While in combat you may spend 3 AP to restore mana equal to half your level, rounded up. After use, cannot be used again until taking a rest.
```

Comment:
This is a must take feat for spell casters.

Reply:
possibly, I might need to change to once every long rest. Originally this was the spellcaster version of second wind.

Reply:
I think this should be tied to spell power, and seems more natural if it's lower regain for each rest. I also like the idea of pushing through and getting mana back at the risk of exhaustion, via a willpower or stamina check.